src/System.Private.CoreLib/shared/System/Runtime/InteropServices/HandleRef.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
 namespace System.Runtime.InteropServices
```

Comment:
`public HandleRef(object wrapper, IntPtr handle)` - wrapper doesn't seem to be guarded against null

Reply:
This is a mistake... both `wrapper` and `_wrapper` should be nullable.  I missed reviewing this file somehow.